src/libpriv/rpmostree-origin.cxx

```diff
@@ -92,13 +92,14 @@ static void
 sync_treefile (RpmOstreeOrigin *self)
 {
   self->treefile.reset ();
-  self->treefile = CXX_MUST_VAL (rpmostreecxx::origin_to_treefile (*self->kf));
+  // Note this may throw a C++ exception
+  self->treefile = rpmostreecxx::origin_to_treefile (*self->kf);
```

Comment:
Note this one - we just now just let a C++ exception get thrown rather than bridging to a GError on which we `assert()`.